### PR TITLE
fix APIs, update norm op, test=develop

### PR DESCRIPTION
**Pull request by JepsonWong**
update norm op for english APIs.

### CODE REVIEW

**Comment by JepsonWong** [comment]
layer_norm
SpectralNorm
LayerNorm
GroupNorm
BatchNorm

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -4253,62 +4253,69 @@ def layer_norm(input,
                act=None,
                name=None):
     """
-    ${comment}
+    **Layer Normalization Layer**
+
+    The OP implements the function of the Layer Normalization Layer and can be applied to mini-batch input data.
```
Comment (JiabinYang) [inline python/paddle/fluid/layers/nn.py]: not OP, API is better

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -4253,62 +4253,69 @@ def layer_norm(input,
...
+    - :math:`x`: the vector representation of the summed inputs to the neurons in that layer.
+    - :math:`H`: the number of hidden units in a layers
+    - :math:`\\epsilon`: the small value added to the variance to prevent division by zero.
+    - :math:`g`: the trainable scale parameter.
+    - :math:`b`: the trainable bias parameter.
 
     Args:
-        input(Variable): The input tensor variable.
-        scale(bool): Whether to learn the adaptive gain :math:`g` after
+        input(Variable): A multidimensional ``Tensor`` of any dimension, and the data type is float32 or float64.
```
Comment (JiabinYang) [inline python/paddle/fluid/layers/nn.py]: multi-dimension

Comment (JiabinYang) [reply python/paddle/fluid/layers/nn.py]: this sentence doesn't make sense, what do u mean about "A multidimensional ``Tensor`` of any dimension"

File: python/paddle/fluid/dygraph/nn.py [hunk 3]
```diff
@@ -1051,89 +1051,98 @@ class BatchNorm(layers.Layer):
...
-        data_layout(string): NCHW|NHWC. Default: NCHW
-        in_place(bool): Make the input and output of batch norm reuse memory. Default: False
-        moving_mean_name(string|None): The name of moving_mean which store the global Mean. Default: None
-        moving_variance_name(string, Default None): The name of the moving_variance which store the global Variance.
-        do_model_average_for_mean_and_var(bool, Default False): Do model average for mean and variance or not.
-        fuse_with_relu (bool): if True, this OP performs relu after batch norm. Default: False
-        use_global_stats(bool): Whether to use global mean and
+        dtype(str, optional): Indicate the data type of the input ``Tensor``,
+             which can be float32 or float64. Default: float32.
+        data_layout(str, optional): Specify the input data format, the data format can be "NCHW" or "NHWC". Default: NCHW.
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py:1113]: By default

File: python/paddle/fluid/dygraph/nn.py [hunk 4]
```diff
@@ -1051,89 +1051,98 @@ class BatchNorm(layers.Layer):
...
+        in_place(bool, optional): Make the input and output of batch norm reuse memory. Default: False.
+        moving_mean_name(str, optional): The name of moving_mean which store the global Mean. Default: None.
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py:1115]: same and all other "Default"

**Review by JiabinYang — COMMENTED** [review]
some comments

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (JepsonWong) [reply python/paddle/fluid/layers/nn.py]: done

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (JepsonWong) [reply python/paddle/fluid/layers/nn.py]: done

File: python/paddle/fluid/dygraph/nn.py (hunk 3 above)
Comment (JepsonWong) [reply python/paddle/fluid/dygraph/nn.py:1113]: done

File: python/paddle/fluid/dygraph/nn.py (hunk 4 above)
Comment (JepsonWong) [reply python/paddle/fluid/dygraph/nn.py:1115]: done

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by wanghaoshuang — APPROVED** [review]

**Review by hutuxian — APPROVED** [review]

**Pull request merged**